arm-network/2016-06-01/network.json

```diff
@@ -6515,14 +6601,143 @@
       ],
       "description": "NetworkSecurityGroup resource"
     },
+	"EffectiveNetworkSecurityGroup": {
+      "properties": {
+        "networkSecurityGroup": {
+	      "$ref": "#/definitions/SubResource",
+          "description": "Gets the id of network security group that is applied"
```

Comment:
Autorest will add "Gets or sets" as a prefix to the description for properties, for languages where appropriate, or just "Gets"  for ReadOnly properties for you, so there is no need to specify them here.

In fact, in this example you can see the description ends up not being very readable:

     "Gets or sets gets the id of network security group that is applied".

Since this isn't identified as ReadOnly it will append "Gets or sets"